### PR TITLE
Fix assertion 'bytes_copied > 0' in JSON.stringify()

### PR DESCRIPTION
**Pull request by rtakacs**
Added a plus condition to the assert that handles the empty string corner case.

### CODE REVIEW

File: tests/jerry/json-stringify.js [hunk 1]
```diff
@@ -14,6 +14,8 @@
 // limitations under the License.
 
 // Checking quoting strings
+assert (JSON.stringify ("") == '""');
```
Comment (zherczeg) [inline tests/jerry/json-stringify.js]: ===

**Comment by zherczeg** [comment]
+1 lgtm

File: tests/jerry/json-stringify.js (hunk 1 above)
Comment (rtakacs) [inline tests/jerry/json-stringify.js]: Fixed.

**Comment by sand1k** [comment]
LGTM.

**Comment by egavrin** [comment]
`make push`

**Comment by egavrin** [comment]
Merged at 292d99ba4e3160590387b3d3d9cedf3a64eb4914
